jrqs-app/src/java/ua/dp/isd/jrqs/SqlBuilder.java

```diff
@@ -69,7 +67,12 @@ public void addCommaSeparatedParameters(String commaSeparatedParameters) {
             return;
         }
 
-        sqlParameters = commaSeparatedParameters.toLowerCase().split(",");
+        String[] splitted = commaSeparatedParameters.toLowerCase().split(",");
+        if (splitted.length > MAX_PARAMETERS) {
+            throw new IllegalArgumentException("The number of maximum allowed arguments have been exceeded.");
```

Comment:
Sounds good for overall restriction. But this particular check is being performed on expanded range mentioned in my previous comment. So, I do not have idea how to extract it from SqlBuilder right now. I had an idea to redo SqlBuilder class. I will think on this during the refactoring. Thank you.